app/forms/Constants.scala

```diff
@@ -17,13 +17,14 @@
 package forms
 
 object Constants {
-  lazy val tirCarnetReferenceMaxLength = 12
-  lazy val maxEoriNumberLength: Int    = 17
-  lazy val minEoriNumberLength: Int    = 14
-  lazy val maxNameLength: Int          = 70
-  lazy val maxTelephoneNumberLength    = 35
-  lazy val minTelephoneNumberLength    = 6
-  lazy val maxRefNumberLength          = 24
-  lazy val maxOtherRefLength           = 35
-  lazy val accessCodeLength            = 4
+  lazy val tirCarnetReferenceMaxLength                      = 12
```

Comment:
A very minor thing but can we please add types to all of these if we are reformatting?

Reply:
Sorted.